android/app/src/main/AndroidManifest.xml

```diff
@@ -56,6 +56,13 @@
             <action android:name="android.intent.action.MAIN" />
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
+
+        <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+          <data android:scheme="italiaapp" android:host="italiaapp" />
```

Comment:
pls document the url schema in the readme, just add a paragraph about it - cc @fpersico

Reply:
also describe the expected format of deep links (as expected by the parsing code below).

Reply:
Added to readme.